velox/functions/prestosql/RowFunction.cpp

```diff
@@ -28,13 +28,18 @@ class RowFunction : public exec::VectorFunction {
       exec::EvalCtx& context,
       VectorPtr& result) const override {
     auto argsCopy = args;
+
+    for (auto& child : args) {
+      VELOX_CHECK_GE(child->size(), rows.end(), "Child size less than Row size")
```

Comment:
> Child size less than Row size

This message is confusing. The check itself is a basic check for an invariant guaranteed by the expression evaluation. apply must ensure that each vector in args has valid values in 'rows'.

Perhaps, a better place for this check would be in expression evaluation just before a call to apply. CC: @bikramSingh91

Reply:
Sg Masha, I will make the change in applyFunction and add this check there..